access-control-service-sql-impl/src/main/java/org/eclipse/tractusx/semantics/accesscontrol/sql/service/SqlBackedAccessControlRuleService.java

```diff
@@ -76,15 +78,24 @@ public List<String> filterValidSpecificAssetIdsForLookup(
    @Override
    public ShellVisibilityCriteria fetchVisibilityCriteriaForShell( ShellVisibilityContext shellContext, String bpn ) throws DenyAccessException {
```

Comment:
Hi,
Just noticed something unfortunately, I think we will need some changes in org.eclipse.tractusx.semantics.accesscontrol.sql.service.SqlBackedAccessControlRuleService#filterValidSpecificAssetIdsForLookup as well after all.